webpack/webpack.dev.babel.js

```diff
@@ -26,6 +27,30 @@ module.exports = (function() {
     optimization: {
       splitChunks: {
         chunks: "all",
+        minSize: 0,
+        minChunks: 1,
+        maxAsyncRequests: 5,
+        maxInitialRequests: Infinity,
```

Comment:
Why are we adjusting the parameters of `minSize`: `0`, `maxAsyncRequests` and `maxInitialRequests` from the defaults?

I feel like `minSize: 0` might start creating chunks that are too small, so the HTTP overhead is bigger than the actual chunk.

And `maxAsyncRequests: 30` and `maxInitialRequests: 30` seem like okay defaults, so we should have some sort of comment saying why we are changing them.

https://webpack.js.org/plugins/split-chunks-plugin/

Reply:
Good catch, I've been experimenting and forgot to change them to sensible values :)